pkg/kubelet/server/server.go

```diff
@@ -66,6 +67,7 @@ import (
 const (
 	metricsPath         = "/metrics"
 	cadvisorMetricsPath = "/metrics/cadvisor"
+	containerHealthPath = "/containerHealth"
```

Comment:
Why is this exposed on a new endpoint, rather than `/metrics`? Minimally I think this should be `/metrics/probes`, but maybe it can just be in metrics?

Reply:
@loburm Did not want this under /metrics. His reasoning was that anything under that endpoint should be metrics about the operation of kubelet only, not about its probes. Maybe we should revisit this discussion?

Reply:
This question was discussed on sig-instrumentation meeting and we have agreed that /metrics endpoint should not expose information about state of containers running on the node. This endpoint contains only kubelet operational metrics and we wanted to keep it like that.

Reply:
Ack. What do you think about `/metrics/probes` then? `/metrics/cadvisor` already has per-container metrics, I believe.

Reply:
I'm fine with /metrics/probes. @loburm does that sound okay?

Reply:
Yeah sure.